app/lib/package/backend.dart

```diff
@@ -1377,30 +1376,22 @@ DerivedPackageVersionEntities derivePackageVersionEntities({
 /// Helper utility class for interfacing with Cloud Storage for storing
 /// tarballs.
 class TarballStorage {
```

Comment:
Should we just do away with this wrapper? Do we need it?

What if we just document the bucket layout when we create an instance of `Bucket`?
```dart
/// Bucket where package tarballs are stored in.
///
/// Organized as follows:
/// ```
/// gs:/<packageBucketName>/packages/<package>-<version>.tar.gz
/// ```
Bucket get packageBucket  => service_scope.... // get instance from service scrope.
```

Or something like that -- I don't recall the actual layout off the top of my head.

But if we write good documentation for the global `packageBucket` getter, then anywhere where you're accessing `packageBucket` you have a good chance find the documentation and check that it aligns with how you are using it.
Then we probably don't need a wrapper that encodes trivial rules.